js/src/components/adaptive-form/adaptive-form-context.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { createContext, useContext } from '@wordpress/element';
+
+/**
+ * @typedef {Object} InputProps
+ * @property {*} value Form value.
+ * @property {boolean} checked Form value converted to boolean.
+ * @property {*} selected Form value.
```

Comment:
This I guess it should be `Selected Form value`. Instead of just `Form value.`

Reply:
It's the same value as `value`. (Its type declaration above is even wrong.)
https://github.com/woocommerce/woocommerce/blob/6.9.0/packages/js/components/src/form/form.tsx#L249-L251